pkg/migration/seed.go

```diff
@@ -12,8 +12,7 @@ import (
 
 func SeedData(ctx context.Context, pending []string, conn *pgx.Conn, fsys fs.FS) error {
 	for _, path := range pending {
-		filename := filepath.Base(path)
-		fmt.Fprintf(os.Stderr, "Seeding data from %s...\n", filename)
+		fmt.Fprintf(os.Stderr, "Seeding data from %s...\n", path)
```

Comment:
Since the seed can now be in multiples locations, I changed to log the path instead of only the filename.

The result will be something like:

```
supabase/seeds/seed1.sql
supabase/another/seed1.sql
```